README.md

```diff
@@ -199,7 +199,7 @@ APIs marked with the `@Deprecated` annotation at the class or method level will
 
 #### io.reactivex.internal.*
 
-All code inside the `io.reactivex.internal.*` packages is considered private API and should not be relied upon at all. It can change at any time. 
+All code inside the `io.reactivex.internal.*` packages are considered private API and should not be relied upon at all. It can change at any time. 
```

Comment:
Yes I think a "the" would solidify it as singular. It's really hard to tell otherwise. I lean towards plural though, but I'm not sure if that's based on what's correct or what I've seen used most.